smithy-model/src/test/resources/software/amazon/smithy/model/loader/invalid/elide-nonexistent-id.smithy

```diff
@@ -0,0 +1,10 @@
+// Member target was elided, but no parent shape contained a matching member name or resource identifier name.
```

Comment:
instead of "parent shape" can we be more explicit and say "no mixin or resource contained a matching"...